docs/docker-example/Dockerfile

```diff
@@ -36,8 +35,12 @@ RUN apt-get install -y vim
 
 # 5) Fix git by reattaching head and making git see other branches than main.
 WORKDIR /workdir/onnx-mlir
+RUN git remote rename origin upstream
```

Comment:
I am not sure why the remote repo needs to be renamed  ?

Reply:
That is because native docker install the "upstream" repo under the "origin" name. But for folks working with their own fork, we typically install our own fork as origin, and the official repo under the "upstream" name.